authors/apps/authentication/tests/test_jwt.py

```diff
@@ -0,0 +1,173 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from rest_framework import status
+from rest_framework.test import APIClient, APIRequestFactory
+
+from authors.apps.authentication.models import User
+from authors.apps.authentication.backends import JWTAuthentication
+
+
+class JWTAuthenticationTest(TestCase):
+    """ Test the JWT Authentication implementation """
+    def setUp(self):
+        self.user_data = {
+            'user': {
+                'email': 'user1@mail.com',
+                'username': 'user1',
+                'password': 'password'
+                }
+            }
+        self.user = User.objects.create(
+            username='user',
+            email='user@mail.com',
+            password='password'
+        )
+        self.login_data = {
+            'user': {
```

Comment:
Considering `self.login_data` is similar to `self.user_data`, You don't need to have a separate dict, you can just use `user_data` when logging in

Reply:
the 2 are not the same, user_data contains the payload required when registering, i.e. username, email and password, whereas, login_data contains the payload requires when logging in i.e email and password